client/lcd/swagger-ui/swagger.yaml

```diff
@@ -323,6 +323,36 @@ paths:
           description: The tx was malformated
         500:
           description: Server internal error
+  /txs/decode:
+    post:
+      tags:
+        - ICS0
```

Comment:
Should be `Transactions` . We renamed all tags as the name convention conflicted with the ICS repo in the cosmos org

Reply:
Thank you !